src/layouts/layouts/SiteViewLayout/SiteViewHeader.tsx

```diff
@@ -29,16 +31,16 @@ export const SiteViewHeader = (): JSX.Element => {
     >
       <HStack spacing="1.25rem">
         <IconButton
-          aria-label="Back to My sites"
+          aria-label="Back to site dashboard"
```

Comment:
Should this label also use the same conditional logic in line 43?

Reply:
yes, good catch!